README.md

```diff
@@ -1,64 +1,213 @@
+
+* [`safe_sign_and_submit_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_and_submit_transaction) — Signs a transaction locally, then submits it to the XRP Ledger. This method does not implement [reliable transaction submission](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission) best practices, so only use it for development or testing purposes.
+
+* [`safe_sign_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_transaction) — Signs a transaction locally. This method **does  not** submit the transaction to the XRP Ledger.
+
+* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method signs a transaction locally, submits the transaction to the XRP Ledger, and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
+
+
+```py
+from xrpl.models.transactions import Payment
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.transaction import send_reliable_submission
+from xrpl.ledger import get_fee
+
+FEE = get_fee(JSON_RPC_CLIENT)
 
-```bash
-poetry run python -m unittest discover tests.integration.reliable_submission
+# prepare the transaction
+my_tx_payment = Payment(
+    account=TESTNET_CLASSIC_ACCOUNT,
+    amount=_AMOUNT,
+    destination=TESTNET_DESTINATION_ACCOUNT,
+    signing_pub_key=TESTNET_WALLET.pub_key,
+    last_ledger_sequence=TESTNET_WALLET.next_sequence_num + 10,
+    sequence=TESTNET_WALLET.next_sequence_num,
+    fee=FEE,
+)
+# sign the transaction
+my_tx_payment_signed = safe_sign_transaction(my_tx_payment,TESTNET_WALLET)
+
+# submit the transaction
+tx_response = send_reliable_submission(my_tx_payment, TESTNET_WALLET, JSON_RPC_CLIENT_TESTNET)
 ```
 
-### Generating Documentation
+### Encode addresses
 
-From the `docs` folder,
+Use the [`xrpl.core.addresscodec`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.core.addresscodec.html) to encode and decode addresses into the "classic" and X-address formats.
 
-```bash
-poetry run sphinx-apidoc -o source/ ../xrpl
-poetry run make html
+```py
+# convert classic address to x-address
+>>> from xrpl.core import addresscodec
+>>> TESNET_XADDRESS = addresscodec.classic_address_to_xaddress(TESTNET_CLASSIC_ACCOUNT, False, True)
+>>> print(TESNET_XADDRESS)
+T7CiH636SkvWbU17fKdttaHE97SXQRW1NfMESd67VDcoZTn
 ```
 
-Sphinx generated docs will be in `docs/_build`.
+## Documentation
+
+For complete reference documentation for the library, see [`xrpl-py` docs](https://xrpl-py.readthedocs.io/en/latest/index.html).
```

Comment:
Per my suggestion above, I suggest moving this up above the "Usage" section so it doesn't end up down here where the tl;dr people don't see it. (Though, those people won't exactly be using it anyway, will they? 😝)